doc/tutorials/dune-package-management/setup.md

```diff
@@ -100,6 +100,13 @@ $ dune exec ./test.exe
 Hello, OCaml, Rust!
 ```
 
+:::{note}
+If you want to only build and fetch the project dependencies, you can use
+`@pkg-install` build alias.
```

Comment:
Nice and good to link up the documentation of the alias, though I would make this into an example, that way people don't have to think what a build alias is:

```suggestion
the `@pkg-install` alias like so

$ dune build @pkg-install
```

(with the right markup)